Multisig/UI/App/UIKit/UI Library/ReusableViews/AddressWithTitleView.swift

```diff
@@ -12,19 +12,61 @@ class AddressWithTitleView: UINibView {
     @IBOutlet private weak var identiconView: UIImageView!
     @IBOutlet private weak var textLabel: UILabel!
     @IBOutlet private weak var detailLabel: UILabel!
+    @IBOutlet private weak var button: UIButton!
+
+    private var style: Style = .nameAndAddress
+    private var address: Address!
+
+    enum Style {
```

Comment:
same question here: maybe by default make this style as address, and only when the name is set to not nil it will be another style?